js/src/base-component.js

```diff
@@ -43,6 +43,14 @@ class BaseComponent {
   static get VERSION() {
     return VERSION
   }
+
+  static get NAME() {
+    throw new Error('You have to implement the static method "NAME", for each component!')
+  }
+
+  static get DATA_KEY() {
+    return `bs.${this.NAME}`
```

Comment:
Wouldn't just `'bs.' + this.Name` be cleaner and shorter?

Reply:
Thank you for our review @alecpl 
I think, we avoid string concatenation, using `+` , so I am not sure which is the 'best'